src/MySTMarkdownCell.tsx

```diff
@@ -32,10 +32,10 @@ export class MySTMarkdownCell
 
   private __rendermime: IRenderMimeRegistry;
 
-  constructor(options: MarkdownCell.IOptions) {
+  constructor(options: MarkdownCell.IOptions, parent: StaticNotebook) {
     super(options);
-
-    this.__rendermime = options.rendermime.clone();
+    // Note we cannot clone this, and it must be the parents (the notebooks)
+    this.__rendermime = parent.rendermime;
```

Comment:
Ah! I didn't get this far in my reading, nice spot.

Reply:
I am not totally sure why this happens, I feel like the clone should work!?

Reply:
This doesn't break anything on my local test build, but I don't think we should be cloning in any case. I think I probably did that in `jupyterlab-imarkdown` in the very early work and never noticed it. We want to use the same rendermime registry as passed in:

```suggestion
    this.__rendermime = options.rendermime;
```